packages/flutter/lib/src/widgets/text.dart

```diff
@@ -372,6 +401,7 @@ class Text extends StatelessWidget {
     this.semanticsLabel,
     this.textWidthBasis,
     this.textHeightBehavior,
+    this.selectionColor = const Color(0xAF6694e8),
```

Comment:
Should this be a constant somewhere?  I wonder where this color came from.  Here and below.